.gitignore

```diff
@@ -1,2 +1,5 @@
 db_config.py
 venv/
+__pycache__/
+.DS_Store
+db_config.py
```

Comment:
i think line 5 should be `Common/db_config.py` ?

Reply:
i think git just matches on the patterns in the .gitignore file, so all db_config.py files will be ignored. i've since removed the duplicate db_config.py entry; i think that was a copy/paste error on my part.